cloud/security.rst

```diff
@@ -126,3 +126,31 @@ All data within Citus Cloud is encrypted at rest, including data on the instance
   <script type="text/javascript">
   analytics.track('Doc', {page: 'Security', section: 'cloud'});
   </script>
+
+Network perimeter controls
+==========================
+
+All Citus clusters run in their own isolated Amazon Virtual Private Cloud (VPC). In addition to all security guarantees which are on by default you can also secure your citus cluster by limiting network access. There are two methods for limiting network access:
```

Comment:
Upcase `citus clusters` to be `Citus clusters`

Reply:
Yeah I'll be giving this pull request an editing pass. Craig opened the first draft for our convenience.